includes/login-register.php

```diff
@@ -107,8 +107,12 @@ function edd_process_login_form( $data ) {
 		// Check for errors and redirect if none present.
 		$errors = edd_get_errors();
 		if ( ! $errors ) {
-			$redirect = apply_filters( 'edd_login_redirect', $data['edd_redirect'], 0 );
-			wp_redirect( $redirect );
+			$default_redirect_url = esc_url_raw( $data['edd_redirect'] );
+			if ( has_filter( 'edd_login_redirect' ) ) {
+				wp_redirect( apply_filters( 'edd_login_redirect', $default_redirect_url ) );
```

Comment:
Probably a good time to add a docblock for this filter: https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters

Reply:
@ashleyfae this filter has actually been in EDD since 1.0, and the `0` we removed used to be a [$user_ID](https://github.com/easydigitaldownloads/easy-digital-downloads/blame/8c3591e346407af71df7d9e857c27b971ed990fc/includes/login-register.php#L98). Should I add it back and note that it's irrelevant? Or can we safely remove the parameter anyway?

Reply:
@robincornett I wonder why it was even removed. :thinking: The user ID is still available via the `$user` object above. Maybe we should just pass that in.